chart/kubeapps/templates/kubeops-rbac.yaml

```diff
@@ -40,5 +40,42 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "kubeapps.kubeops.fullname" . }}
     namespace: {{ .Release.Namespace }}
+{{- if .Values.allowNamespaceDiscovery }}
```

Comment:
I don't see it disappearing, no, it was more so we can evaluate it and decide on a default before exposing it as a top-level flag. But I agree it can be managed either way.